bin/generic_plot_script.R

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env Rscript
+
+library(ggplot2)
+library(gplots)
+library(reshape2)
+
+#parse commands
+args <- commandArgs()
+Sample <- args[6]
+#Condition<- args[2]
+
+
+
+
+# plot library RT deviation
```

Comment:
maybe better name for script? a bit to generic

Reply:
changed the name to plot_quantities_and_counts.R